kuma/static/styles/includes/mixins.styl

```diff
@@ -50,6 +50,13 @@ set-larger-font-size() {
     set-font-size($base-bump-font-size);
 }
 
+media-query-emify(value) {
+    /* media queries work of browser base font size (16px),
+       even if we change html font size */
+    v = unit(value / 16px, 'em');
```

Comment:
Would it make sense to use `$base-font-size` here instead of `16px`? Or is *our* base font size irrelevant?